dataactcore/utils/jobQueue.py

```diff
@@ -34,7 +34,7 @@ def enqueue(jobID):
             response = requests.post(url, params)
             return response.json()
 
-        @self.jobQueue.task(name='jobQueue.generate_d1')
+        @self.jobQueue.task(name='jobQueue.generate_d_file')
```

Comment:
Just to make sure I'm understanding this correctly, it looks like the function below takes which D file to generate as an argument?